readme.md

```diff
@@ -76,25 +267,44 @@ var Hello = React.createClass({
 })
 ```
 
-1. `var Hello`
-  * This is the component we're creating. In this example, we are creating a "Hello" component.
+#### What's that HTML doing in my Javascript?
+
+Often times we write out React components in **JSX**.
+* JSX is an alternate Javascript syntax that allows us to write code that strongly resembles HTML. It is eventually compiled to lightweight JavaScript objects.
+* React then uses these objects to build out a "Virtual DOM" -- more on that in just a bit.
+
+> React can be written without JSX. If you want to learn more, [check out this blog post](http://jamesknelson.com/learn-raw-react-no-jsx-flux-es6-webpack/).  
+
+Let's break down the things we see here...
 
-2. `React.createClass`
-  * This is the React library method we use to create our component definition.
+##### `var Hello`
+This is the component we're creating. In this example, we are creating a "Hello" component.
+
+##### `React.createClass`
+This is the React library method we use to create our component definition.
   * Takes an object as an argument.
 
-3. `render`
-  * Every component has, at minimum, a render method.
-  * Generates a Virtual DOM node that will be added to the actual DOM.
-    * What does a Virtual DOM node sound like it does?
-    * Looks just like a regular ol' DOM node, but it's not yet attached to the DOM.
-    * Instead, it acts as a staging area for changes that will eventually be implemented.
-  * The contents of this node are defined in the method's return statement.
-    * At the moment, this return value looks exactly like HTML. But it's not...
+##### `render`
+Every component has, at minimum, a render method. It generates a **Virtual DOM** node that will be added to the actual DOM.
+* Looks just like a regular ol' DOM node, but it's not yet attached to the DOM.
+
+#### Virtual DOM? How is that different from the usual DOM?
+
```

Comment:
could be helpful to include some diagrams here....from http://blog.reverberate.org/2014/02/react-demystified.html